plugins/api-docs/src/components/ComponentsCards/ComponentsTable.tsx

```diff
@@ -14,23 +14,58 @@
  * limitations under the License.
  */
 
-import { ComponentEntity } from '@backstage/catalog-model';
+import {
+  ComponentEntity,
+  EntityName,
+  RELATION_OWNED_BY,
+  RELATION_PART_OF,
+} from '@backstage/catalog-model';
 import { Table, TableColumn } from '@backstage/core';
+import {
+  EntityRefLink,
+  EntityRefLinks,
+  formatEntityRefTitle,
+  getEntityRelations,
+} from '@backstage/plugin-catalog-react';
 import React from 'react';
-import { EntityLink } from '../EntityLink';
 
-const columns: TableColumn<ComponentEntity>[] = [
+type EntityRow = ComponentEntity & {
+  row: {
+    partOfSystemRelationTitle?: string;
+    partOfSystemRelations: EntityName[];
+    ownedByRelationsTitle?: string;
+    ownedByRelations: EntityName[];
+  };
+};
+
+const columns: TableColumn<EntityRow>[] = [
   {
     title: 'Name',
     field: 'metadata.name',
     highlight: true,
-    render: (entity: any) => (
-      <EntityLink entity={entity}>{entity.metadata.name}</EntityLink>
+    render: entity => (
+      <EntityRefLink entityRef={entity}>{entity.metadata.name}</EntityRefLink>
+    ),
+  },
+  {
+    title: 'System',
+    field: 'row.partOfSystemRelationTitle',
+    render: entity => (
+      <EntityRefLinks
+        entityRefs={entity.row.partOfSystemRelations}
+        defaultKind="system"
+      />
     ),
   },
   {
     title: 'Owner',
-    field: 'spec.owner',
+    field: 'row.ownedByRelationsTitle',
+    render: entity => (
+      <EntityRefLinks
+        entityRefs={entity.row.ownedByRelations}
```

Comment:
the naming of the fields gets a bit odd :) we could have had `type EntityRow = { entity: ComponentEntity, extra: {...} }` so then we'd render `<EntityRefLinks entityRefs={row.extra.ownedByRelations` etc

Reply:
That is neat, I like that idea! Updated it 🙂